src/main/java/net/mcreator/element/parts/gui/EntityModel.java

```diff
@@ -62,15 +60,10 @@ public EntityModel(int x, int y, Procedure entityModel, Procedure displayConditi
 		return -1;
 	}
 
-	public int getID(EntityModel entityModel) {
-		int id = 0;
-		for (GUIComponent component : CustomGUIGUI.staticComponentList.getComponentList()) {
-			if (component instanceof EntityModel model) {
-				if (model.equals(entityModel))
-					break;
-				id++;
-			}
-		}
-		return id;
+	public String procedureName(String name) {

```

Comment:
You can simply use `RegistryNameFixer.fromCamelCase()` instead of having to write another method.